docs/README.md

```diff
@@ -8,34 +8,34 @@
 
-| Block name                    | Description                                                                                                     |
-| ----------------------------- | --------------------------------------------------------------------------------------------------------------- |
-| `product-specification-group` | Renders the product specification group.                                                     |
-| `product-specification`       | Renders the product specification. It should be declared as a children of `product-specification-group`. |
-| `product-specification-value` | Renders the product specification value. It should be declared as a children of `product-specification`. It is possible to render with `HTML`. |
-| `product-specification-text`  | Mandatory children of `product-specification-group`, `product-specification` and `product-specification-value`. Depending on which block it is declared, the `product-specification-text` renders data regarding a specification group, a specification, or a specification value. |
+| Block name                    | Description                                                                                                                                                                                                                                                                             |
+| ----------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `product-specification-group` | Renders the product specification group.                                                                                                                                                                                                                                                |
+| `product-specification`       | Renders the product specification. It should be declared as a child of `product-specification-group`.                                                                                                                                                                                   |
+| `product-specification-value` | Renders the product specification value. It should be declared as a child of `product-specification`. It can be rendered with `HTML`.                                                                                                                                                   |
+| `product-specification-text`  | Mandatory children of `product-specification-group`, `product-specification`, and `product-specification-value`. Depending on which block is declared, the `product-specification-text` renders information regarding a specification group, a specification, or a specification value. |
```

Comment:
```suggestion
| Block name | Description |
| - | - |
| `product-specification-group` | Renders the product specification group. |
| `product-specification` | Renders the product specification. It should be declared as a child of `product-specification-group`. |
| `product-specification-value` | Renders the product specification value. It should be declared as a child of `product-specification`. It can be rendered with `HTML`. |
| `product-specification-text`  | Mandatory children of `product-specification-group`, `product-specification`, and `product-specification-value`. Depending on which block is declared, the `product-specification-text` renders information regarding a specification group, a specification, or a specification value. |
```